app/models/minutes.js

```diff
@@ -0,0 +1,8 @@
+import Piece from './piece';
+import { attr, belongsTo } from '@ember-data/model';
+
+export default class Report extends Piece {
```

Comment:
`class Minutes` ?

Reply:
This was fixed in another branch, will fix here as well

Reply:
fixed in [f8da6c6](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1737/commits/f8da6c60a6961df6793899f8ab321b7dcc6b6d4b)